src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/constantname/Example1.java

```diff
@@ -1,10 +1,5 @@
 /*
 ConstantName
-format = (default)^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$
```

Comment:
Why ? 
We need this so much.
We need to update check config that complain on such lines

Reply:
You said in discord that we should not include default properties.

Reply:
I meant, no defaults in xml format. But in properties format ok to use, as it is not publicly visible.

Reply:
@romani Converting configs to `/*xml <module name="..` is part of the plan, right? I don't see a reason to keep the default props.

Reply:
Yes, you are right, it will keep our file size shorter.

Reply:
I agree to not having defaults.